wcfsetup/install/files/acp/templates/multifactorAuthentication.tpl

```diff
@@ -0,0 +1,44 @@
+{capture assign='pageTitle'}{lang}wcf.user.security.multifactor.authentication{/lang}{/capture}
+{capture assign='contentTitle'}{lang}wcf.user.security.multifactor.authentication{/lang}{/capture}
+
+{include file='header' __isLogin=true}
+
+<section class="box">
+		<h2 class="boxTitle">{lang}wcf.user.security.multifactor.methods{/lang}</h2>
+		
+		<div class="boxContent">
+			<nav>
+				<ol class="boxMenu">
+					{foreach from=$setups item='_setup'}
+						<li{if $setup->getId() == $_setup->getId()} class="active"{/if}>
+							<a class="boxMenuLink" href="{link controller='MultifactorAuthentication' object=$_setup url=$redirectUrl}{/link}"><span class="boxMenuLinkTitle">{lang}wcf.user.security.multifactor.{$_setup->getObjectType()->objectType}{/lang}</span></a>
+						</li>
+					{/foreach}
+				</ol>
+			</nav>
+		</div>
+	</section>
```

Comment:
This indentation looks off by one level.

Reply:
Ah, yes indeed, I'll fix that. However this page needs a full relayout anyway, because I simply copied over the frontend HTML which isn't really compatible with the ACP's CSS.